lib/features/mon_suivi/mon_suivi_actions.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:pass_emploi_app/models/date/interval.dart';
+import 'package:pass_emploi_app/models/mon_suivi.dart';
+
+enum Period {
+  previous,
+  current,
+  next;
+
+  bool isCurrent() => this == Period.current;
```

Comment:
Tu peux aussi faire un getter pour ce genre de truc 
`bool get isCurrent => this == Period.current;

Reply:
Bien vu ;) J'y pense jamais…